docs/setup/aws.md

```diff
@@ -21,20 +21,8 @@ Grapl comes with binaries already in the repository.
 
 Clone the repo:
 `git clone https://github.com/insanitybit/grapl.git`
-`cd ./grapl/grapl-cdk/`
-`npm i # install dependencies`
-`cdk boostrap # set up aws-cdk`
-
-Add a `.env` file, and fill it in:
-
-`BUCKET_PREFIX="<unique prefix to differentiate your buckets>"`
-
-Run the deploy script
-`./deploy_all.sh`
-
-It will require confirming some changes to security groups, and will take a few minutes to complete.
-
-This will give you a Grapl setup that’s adequate for testing out the service.
+`cd grapl/src/js/grapl-cdk/`
+`<Follow the instructions in the README.md>`
```

Comment:
unfortunately, the instructions in this file didn't mention the dgraph provisioning. I want to deduplicate documentation, so, I'm just directing readers to the cdk readme.